megamek/src/megamek/client/ui/swing/boardview/BoardView.java

```diff
@@ -5239,9 +5239,12 @@ public void updateEcmList() {
             }
 
             // hidden enemy entities don't show their ECM bubble
-            if (ecmInfo.getEntity().getOwner().isEnemyOf(localPlayer) && ecmInfo.getEntity().isHidden()) {
-                continue;
-            }
+                if (ecmInfo.getEntity() != null
```

Comment:
Bug I encountered here when I create an ECM field without an entity tied to it, this hidden entity check will cause a null pointer exception. I confirmed this happens with both the new Chaff pods and the Vehicular Grenade Launcher chaff grenades.


Not sure if additional work is needed to hide ECM fields without an entity associated with them.

Reply:
Don't know either. Let's see what happens.